eclair-core/src/main/scala/fr/acinq/eclair/channel/fsm/Channel.scala

```diff
@@ -1073,6 +1084,22 @@ class Channel(val nodeParams: NodeParams, val wallet: OnChainChannelFunder, val
 
     case Event(BITCOIN_FUNDING_TIMEOUT, d: DATA_CLOSING) => handleFundingTimeout(d)
 
+    case Event(w: WatchFundingConfirmedTriggered, d: DATA_CLOSING) =>
+      d.alternativeCommitments.find(_.commitInput.outPoint.txid == w.tx.txid) match {
+        case Some(alternativeCommitments) =>
+          log.info("an alternative funding tx with txid={} got confirmed", w.tx.txid)
```

Comment:
```suggestion
          log.info("an alternative funding tx with txid={} got confirmed", w.tx.txid)
          // This is a corner case where:
          // - we are using dual funding
          // - *and* the funding tx was RBF-ed
          // - *and* we went to CLOSING before any funding tx got confirmed (probably due to a local or remote error)
          // - *and* an older version of the funding tx confirmed and reached min depth (it won't be re-orged out).
          //
          // This means that:
          // - the whole current commitment tree has been double-spent and can safely be forgotten
          // - from now on, we only need to keep track of the commitment associated to the funding tx that got confirmed
```

Reply:
Added in https://github.com/ACINQ/eclair/pull/2274/commits/7b7bfac5bb44092cac0a7439723abff63ae4f356